yorkie/src/main/kotlin/dev/yorkie/document/crdt/CrdtArray.kt

```diff
@@ -10,14 +10,21 @@ internal class CrdtArray(
     createdAt: TimeTicket,
 ) : CrdtContainer(createdAt), Iterable<CrdtElement> {
     val head
-        get() = elements.dummyHead.value
+        get() = elements.getHead()
 
     val last
         get() = elements.last.value
 
     val length
         get() = elements.length
 
+    /**
+     * Returns the creation time of the last element.
+     */
+    fun getLastCreatedAt(): TimeTicket {
```

Comment:
Although it is only my personal preference, I would use property rather than functions when it always returns the same instance and has no parameters.
(same for `RgaTreeList.getHead`)

Reply:
If there are no problems, it would be better to unify the style. The proposed method is looking neater.